app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -125,6 +127,16 @@ public void onProgressChanged(WebView view, int newProgress) {
             return false;
         });
         setWebViewSettings(mWebView.getSettings());
+        mSwipeContainer = (SwipeRefreshLayout) view.findViewById(R.id.swipe_container);
+        mSwipeContainer.setColorSchemeResources(R.color.white);
+        mSwipeContainer.setProgressBackgroundColorSchemeResource(
+            AppUtils.getThemedResId(getActivity(), R.attr.colorAccent));
+        mSwipeContainer.setOnRefreshListener(() -> {
+            WebSettings webSettings = mWebView.getSettings();
+            webSettings.setCacheMode(WebSettings.LOAD_NO_CACHE);
```

Comment:
This should come after `setWebViewSettings()` which set cache mode to `LOAD_CACHE_ELSE_NETWORK`